src/app/static/src/app/utils.ts

```diff
@@ -54,3 +54,21 @@ export const verifyCheckSum = (content: string): false | any => {
 
     return valid && JSON.parse(data);
 };
+
+export const freezer = {
+
+    deepFreeze : (data: any): any => {
+        if (Array.isArray(data)) {
```

Comment:
Will this not do a shallow freeze on the array items, when we probably need to deep freeze those too? (e.g. gejson features, I think).

Reply:
Oh yeah, you're right, I misunderstood the documentation on this.